Add support for antialias and related subpixel order settings

jhaygood86 opened the pull request:
Example screenshot:

![Screenshot from 2021-09-04 19-48-13](https://user-images.githubusercontent.com/360393/132110597-ed965ae8-f8b3-46a3-b4db-9b8cfd1d96bb.png)

Fixes #228

kdwk commented:
I think the spacing between the newly added options should be increased, because now the options seem too closely packed

jhaygood86 commented:
![Screenshot from 2021-09-05 19 10 06](https://user-images.githubusercontent.com/360393/132143880-a95bce62-671d-430f-b1f4-12f3cee91aaf.png)

Added some margins between the new settings.

cassidyjames commented:
My inclination with these additional text-related options is that we should move this layout over to a sidebar with:

- Wallpaper
- Appearance (or Style?)
- Text (is that the best name?)
- Dock & Panel
- Multitasking

Does that make sense @elementary/ux?

danrabbit commented:
@cassidyjames yeah ultimately moving to a sidebar would be better imo. For the purposes of this PR, it would probably be good just to split the text options off into another tab

I'd also be inclined to use radios for these options instead of modebuttons

jhaygood86 commented:
![Screenshot from 2021-09-09 08-14-22](https://user-images.githubusercontent.com/360393/132683845-a0198dc7-9906-4fd9-b3e5-23c0e5b8d170.png)

Added a new "Text" tab and moved text related settings there. Switched mode buttons to radio buttons for new settings

cassidyjames reviewed (changes requested):
Couple of design thoughts in no particular order (I'll also comment inline in the code where it makes sense and/or make a PR off this one):

- Is there a valid reason to use "None" on modern hardware?
- Perhaps we could simplify this to "Default" and "Subpixel" to be a _little_ less technical; we don't really have to explain how anti-aliasing and subpixels work since users can choose it based on looks.
- I wonder if we should include a longer-form example in a 
- We could probably use an Infobar that appears on change to inform that apps will need to be re-opened to see changes, like we do in other plugs. This would clean up the amount of help copy.
- Labels on the left should be top-aligned.
- We probably want to leave the subpixel settings visible but insensitive when unused so they don't seemingly randomly appear and change the flow of the page.
- We can probably drop "Text" from each of these labels since the whole page is about "Text"
- I wonder if a horizontal layout would work better for these radio options to take up less space?
- I wonder if these should be under their own subheading with an example? Let me prototype that up :smile:
